src/main/java/com/angelbroking/smartapi/smartstream/ticker/SmartStreamTicker.java

```diff
@@ -63,12 +65,36 @@ public SmartStreamTicker(String clientId, String feedToken, SmartStreamListener
         init();
     }
 
+	/**
+	 * Initializes the SmartStreamTicker.
+	 *
+	 * @param clientId            - the client ID used for authentication
+	 * @param feedToken           - the feed token used for authentication
+	 * @param delay               - delay in milliseconds
+	 * @param period              - period in milliseconds
+	 * @param smartStreamListener - the SmartStreamListener for receiving callbacks
+	 * @throws IllegalArgumentException - if the clientId, feedToken, or SmartStreamListener is null or empty
+	 */
+	public SmartStreamTicker(String clientId, String feedToken, SmartStreamListener smartStreamListener, Integer delay, Integer period ) {
+		if (clientId.isEmpty() || feedToken.isEmpty() || Utils.isEmpty(delay) || Utils.isEmpty(period) ||  Utils.validateInputNullCheck(smartStreamListener)) {
+			throw new IllegalArgumentException(
+					"clientId, feedToken and SmartStreamListener should not be empty or null");
+		}
+		this.DELAY_IN_SECONDS = delay;

```

Comment:
if Delay is classs level variable then use small case format.

Reply:
Changes done, now the class level variables are in small cases format only.